.github/workflows/smoke-windows-mingw64.yml

```diff
@@ -39,4 +39,5 @@ jobs:
         run: |
             cd win32
             set HARNESS_OPTIONS=j2
+            set CCHOME=C:\strawberry\c\bin
```

Comment:
It needs to be `C:\strawberry\c`, without `bin`.

Reply:
Fixed in the latest revision, which now is all-green for me